src/components/EnvironmentOverride/SecretOverrides.tsx

```diff
@@ -282,7 +319,14 @@ export function OverrideSecretForm({ name, toggleCollapse }) {
                     return agg
                 }, {})
             }
-            dispatch({ type: 'loadingSubmit' })
+            if (type === 'volume') {
+                payload['mountPath'] = state.mountPath;
+                payload['subPath'] = state.subPath;
+                if (isFilePermissionChecked) {
+                    payload['filePermission'] = state.filePermission.value.length <= 3 ? `0${state.filePermission.value}` : `${state.filePermission.value}`;
```

Comment:
dont prepend 0 in the text box itself.. only add it before sending

Reply:
That is what I am doing here, adding it to payload before sending